djweb/simulations/static/js/assembly_addition.js

```diff
@@ -1,6 +1,12 @@
 const computer_size = 7;
 const num_code_lines = 4;
 const highlighted_registers = [6];
+const line_descriptions = [
```

Comment:
I really appreciate this addition. However, I think the current organization structure is too complicated. 
Would it be possible to combine both the symbolic code and the descriptions into a Program class? 
Within the program class, each line could have both code, descriptions, and possibly highlight instructions (e.g. when executing a line that deals with the accumulator, adding an instruction to highlight the accumulator).

Reply:
That sounds useful.  @felixtran39 is working on changing the interface right now, maybe you guys should talk and we can put in a future pull request?  My concern is that Felix and I both changing the GUI at the same time could be problematic.